### DISCUSSION TITLE
Possible Good First Issues

### DISCUSSION THREAD

**Discussion by Asdf0717** [facebook/docusaurus#5417] [General]
Hello, we are a group of researchers developing machine learning techniques to locate issues suitable for newcomers, and our model considers the following issues as likely "good first issues"

#4823 #5379 #5380 #5400 

May we recommend you to label them as "good first issue" so newcomers know where to choose? 

Thank you!

**Comment by slorber** [comment]
Hey

On the list above, I would only qualify the last one as a good first issue (#5400).
Others are likely a bit more tricky, eventually requiring advice from maintainers
